### PR TITLE
[localization] Translate login and signup page in Spanish #4180

### LINKED ISSUE #4180: [localization] Translate login and signup page in Spanish
### Is your proposal related to a problem?

Add translations for the login and signup page in Spanish

### Describe the solution you'd like

The strings need to be translated:

Screenshot 2022-09-30 at 3 38 25 PM (1)

**Here's the guide to contributing to [localization](https://docs.tooljet.com/docs/contributing-guide/l10n)**

### PR DESCRIPTION
**Pull request by Azanul**
fixes #4180

### CODE REVIEW

**Comment by Azanul** [comment]
@withshubh Please add relevant tags and assign reviewers. Do let me know if I'm missing anything.

File: frontend/assets/translations/es.json [hunk 1]
```diff
@@ -0,0 +1,921 @@
...
+        "enterBusinessEmail":"Ingrese el correo electrónico de su empresa",
+        "alreadyHaveAnAccount":"¿Ya tiene una cuenta?",
+        "password":"Contraseña",
+        "showPassword":"Mostrar contraseña",
+        "loginTo":"Acceder a",
+        "yourAccount":"su cuenta",
+        "noLoginMethodsEnabled":"No hay métodos de inicio de sesión habilitados para este espacio de trabajo",
+        "emailConfirmLink":"Compruebe su correo electrónico para ver el enlace de confirmación por favor",
+        "newPassword":"Nueva Contraseña",
+        "passwordConfirmation":"Confirmación de contraseña"
```
Comment (withshubh) [inline frontend/assets/translations/es.json:91]: LGTM

**Review by withshubh — APPROVED** [review]

File: frontend/assets/translations/es.json [hunk 2]
```diff
@@ -0,0 +1,921 @@
...
+        "dontHaveAccount":"¿Aún no tienes una cuenta?",
+        "signIn":"Acceder",
+        "signUp":"Registrarte",
+        "createToolJetAccount":"Crear una cuenta de ToolJet",
+        "enterBusinessEmail":"Ingrese el correo electrónico de su empresa",
+        "alreadyHaveAnAccount":"¿Ya tiene una cuenta?",
+        "password":"Contraseña",
+        "showPassword":"Mostrar contraseña",
+        "loginTo":"Acceder a",
+        "yourAccount":"su cuenta",
```
Comment (LPRegen) [inline frontend/assets/translations/es.json:87]: Hi, just a small detail here. To keep the consistency between the values I would recommend to replace the first character for a capital one `Su cuenta`.

The rest of the translations are great!

**Review by LPRegen — COMMENTED** [review]
There is one small detail on [line 87](https://github.com/ToolJet/ToolJet/pull/4324/files#r996672848) the rest of the work is excellent.

**Comment by withshubh** [comment]
Hey @Azanul 👋 
Let's update the changes suggested by @LPRegen.

File: frontend/assets/translations/es.json (hunk 2 above)
Comment (Azanul) [reply frontend/assets/translations/es.json:87]: I don't know how I missed it. Thanks for pointing it out.

**Comment by Azanul** [comment]
> Hey @Azanul wave Let's update the changes suggested by @LPRegen.

Done

**Review by withshubh — APPROVED** [review]

**Review by LPRegen — COMMENTED** [review]
The requested change has been applied.

Again, excellent work!

**Comment by Navaneeth-pk** [comment]
@Azanul please resolve the merge conflicts.

**Comment by Azanul** [comment]
@Navaneeth-pk Done

**Review by Navaneeth-pk — APPROVED** [review]

**Comment by withshubh** [comment]
Hey @Azanul 
Thanks for contributing to ToolJet. :raised_hands: 
Please submit the form to claim your Hacktoberfest swag: https://airtable.com/shr1qR6xhqsWO7LD7

**Pull request merged**